tests/pageObject/loginTermsPage.page.js

```diff
@@ -0,0 +1,38 @@
+// loginTermsPage.js
+
+const Page = require('./page.page')
+
+class LoginTermsPage extends Page {
+  open() {
+    super.open('/wallets')
+  }
+  get h1() {
+    return $('h1')
+  }
+
+  get checkboxes() {
+   return [
+    $('#root > div > div > div > div:nth-child(1) > div > div > div:nth-child(2) > div:nth-child(1) label'),
+    $('#root > div > div > div > div:nth-child(1) > div > div > div:nth-child(2) > div:nth-child(2) label'),
+    $('#root > div > div > div > div:nth-child(1) > div > div > div:nth-child(2) > div:nth-child(3) label'),
+     //$('#root > div > div > div > div:nth-child(1) > div > div > div:nth-child(2) > div:nth-child(3)'),
+    $('#root > div > div > div > div:nth-child(1) > div > div > div:nth-child(2) > div:nth-child(4) label'),
+    ]
+  }
+
+  get termsOfUseLink() {
+    return $('#root > div > div > div > div:nth-child(1) > div > div > div:nth-child(2) > div:nth-child(4) > div > label > a:nth-child(5)')
+  }
+
+  get privacyPolicyLink() {
+    return $('#root > div > div > div > div:nth-child(1) > div > div > div:nth-child(2) > div:nth-child(4) > div > label > a:nth-child(7)')
+  }
+
+  get confirmAndContinueButton() {
+    //return $('#root > div > div > div > div:nth-child(1) > div > div > div._17RbzQLX')
```

Comment:
Would be great to have a description comment about why do we have this commented selector

Reply:
This delimiter will be deleted at the next  PR, thanks!